tests/test_subdivision.py

```diff
@@ -164,7 +164,7 @@ def test_subdivision_solve_with_transform_1d():
     #Case 6 - One MultiPower 1D of degrees 10
     #choose a seed that has a zero like ?
     np.random.seed(2)
-    a = -2*np.ones(1);b = 2*np.ones(1)
+    a = -1.1*np.ones(1);b = 1.1*np.ones(1)
```

Comment:
Going from -2,2 caused problems with the way this test is written. Essentially the values get large and so with the relative tolerance being used it isn't solved with an abs tolerance of less than 1e-3 which is what the test is measuring